x/checkers/keeper/msg_server_test.go

```diff
@@ -4,13 +4,20 @@ import (
 	"context"
 	"testing"
 
-	keepertest "github.com/b9lab/checkers/testutil/keeper"
-	"github.com/b9lab/checkers/x/checkers/keeper"
-	"github.com/b9lab/checkers/x/checkers/types"
+	keepertest "github.com/b9blab/checkers/testutil/keeper"
+	"github.com/b9blab/checkers/x/checkers/keeper"
+	"github.com/b9blab/checkers/x/checkers/types"
```

Comment:
b9blab? That cannot be 🤔

Reply:
Ops sorry :)

I modified it manually since locally I have "alice" instead of "b9lab".

In general I added the test to remove the "warning" about unused function

Reply:
Can you fix it and I will try my cherry-picking and squashing?